schema.json

```
@@ -1202,41 +1292,62 @@
                     "uniqueItems": true
                 },
                 "end": {
+                    "description": "The end page of the work.",
                     "type": "integer"
                 },
                 "entry": {
+                    "description": "An entry in the collection that constitutes the work.",
                     "type": "string"
                 },
                 "filename": {
+                    "description": "The name of the electronic file containing the work.",
                     "type": "string"
                 },
                 "format": {
+                    "description": "The format in which a work is represented.",
                     "type": "string"
                 },
+                "identifiers": {
+                    "description": "The persistent identifier(s) of the work (for identifiers that are not DOIs).",
```

```suggestion
                    "description": "The persistent identifier(s) of the work.",
```

See comment above for identifiers.